docker-compose.yml

```diff
@@ -28,13 +28,15 @@ services:
       # if you develop on Linux, you may use a bind-mounted host directory instead
       # - ./public/media:/srv/sylius/public/media:rw
       - public-media:/srv/sylius/public/media:rw
+      - sylius-vendor:/srv/sylius/vendor
+      - sylius-node_modules:/srv/sylius/node_modules
 
   mysql:
     image: percona:5.7
     environment:
       - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD:-nopassword}
-      - MYSQL_DATABASE=sylius
-      - MYSQL_USER=sylius
+      - MYSQL_DATABASE=sylius_${APP_ENV}
+      - MYSQL_USER=${MYSQL_ROOT_PASSWORD:-sylius}
```

Comment:
Unedited copy/paste?

Reply:
Nice catch! my fault to not review it before pull